apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -411,6 +421,13 @@ if [[ "$should_use_xcodebuild" == true ]]; then
     "%(xctestrun_template)s" > "$xctestrun_file"
 
 
+if [[ -n "${DEBUG_XCTESTRUNNER:-}" ]]; then
+  echo
+  echo "xctestrun contents:"
+  cat "$xctestrun_file"
+  echo
+fi
+
```

Comment:
Please update the indentation:
```suggestion
  if [[ -n "${DEBUG_XCTESTRUNNER:-}" ]]; then
    echo
    echo "xctestrun contents:"
    cat "$xctestrun_file"
    echo
  fi

```